### PR TITLE
Set 'Consider only...' checkbox by default if both app-level settings are enabled.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
This is what the new warnings look like (they must be acknowledged before the create/edit application dialogs can be submitted):
image
image

**Comment by duftler** [comment]
@ajordens and @danielpeach please review.

**Comment by ajordens** [comment]
Can we only show the warning when the user changes the instance health settings (vs when the application is created?)

File: app/scripts/modules/titus/serverGroup/configure/ServerGroupCommandBuilder.js [hunk 1]
```diff
@@ -40,6 +40,10 @@ module.exports = angular.module('spinnaker.titus.serverGroupCommandBuilder.servi
         }
       };
 
+      if (application.attributes && application.attributes.platformHealthOnlyShowOverride && application.attributes.platformHealthOnly) {
+        command.interestingHealthProviderNames = ['Titus'];
```
Comment (ajordens) [inline app/scripts/modules/titus/serverGroup/configure/ServerGroupCommandBuilder.js]: I'm not sure we actually need this for `titus`.

Comment (duftler) [inline app/scripts/modules/titus/serverGroup/configure/ServerGroupCommandBuilder.js]: Backed out these changes for titus.

**Comment by duftler** [comment]
Reworded the message when enabling showOverrideWarning as discussed offline.

**Comment by duftler** [comment]
Relates to https://github.com/spinnaker/deck/pull/2615 and https://github.com/spinnaker/orca/pull/957.

**Pull request merged**